apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -180,14 +180,14 @@ const AppearanceView = () => {
                   <p className="font-semibold ltr:mr-2 rtl:ml-2">
                     {t("disable_cal_branding", { appName: APP_NAME })}
                   </p>
-                  <Badge variant="gray">{t("pro")}</Badge>
+                  {!ENABLE_PAID_FEATURES && <Badge variant="gray">{t("pro")}</Badge>}
```

Comment:
We don't use "PRO" plans anymore so this can be safely removed
```suggestion
```

Reply:
not so fast, this is referring to the new TEAM plan

Reply:
you still need a TEAM plan for disabling branding

Reply:
I mean that we don't use the PRO Badge anymore @PeerRich

Reply:
right but we should still show a badge that an upgrade is required @Jaibles what should the badge say?

Reply:
We're currently using an "Upgrade" badge in other places with a tooltip.
<img width="935" alt="CleanShot 2023-01-10 at 16 33 01@2x" src="https://user-images.githubusercontent.com/4536123/211608497-b371532e-6cb1-4fdd-affd-23c817f9d81f.png">

Reply:
Change the copy of the tooltip to:

"You need to upgrade to a TEAM plan to use this feature. Click here and create a team."